tests/cc5/input.dat

```diff
@@ -1,41 +0,0 @@
-#! RHF CCSD(T) aug-cc-pvtz frozen-core energy of C4NH4 Anion
-
-memory 12 gb
-
-refnuc   =  135.0921284884196    #TEST
-refscf   = -208.2159793502       #TEST
-refccsd  =   -0.884280512675734  #TEST
-ref_t    =   -0.048772959402886  #TEST
-reftotal = -209.149032822278599  #TEST
```

Comment:
copy this bit commented into the `cc5a` file, with a comment `aug-cc-pvtz`, so it's not lost if someone wants to run it again by hand.